yarn.lock

```diff
@@ -8093,7 +8093,7 @@ goober@^2.0.39:
   resolved "https://registry.yarnpkg.com/goober/-/goober-2.1.8.tgz#e592c04d093cb38f77b38cfcb012b7811c85765e"
   integrity sha512-S0C85gCzcfFCMSdjD/CxyQMt1rbf2qEg6hmDzxk2FfD7+7Ogk55m8ZFUMtqNaZM4VVX/qaU9AzSORG+Gf4ZpAQ==
 
-goober@^2.0.41, goober@^2.0.5:
+goober@^2.0.41, goober@^2.0.5, goober@^2.1.11:
```

Comment:
double check if 2.1 has any breaking changes (shouldn't, in theory) over 2.0

Reply:
Doesn't look like any breaking changes were introduced with the launch of 2.1 (just reduction in overall package size) so I think we're fine